src/NuGetPackageVerifier/WinTrust.cs

```diff
@@ -93,23 +93,24 @@ public static bool IsAuthenticodeSigned(string path)
         {
             var fileInfo = new WinTrustFileInfo()
```

Comment:
We're generally not going to be Authenticode signing in dev workflows. So it might be fine to explicitly fail here if we're not on Windows.